src/pydicom/pixels/processing.py

```diff
@@ -974,20 +1081,52 @@ def _convert_YBR_FULL_to_RGB(arr: "np.ndarray") -> "np.ndarray":
     """
     orig_dtype = arr.dtype
 
-    ybr_to_rgb = np.asarray(
-        [
-            [1.000, 1.000, 1.000],
-            [0.000, -0.114 * 1.772 / 0.587, 1.772],
-            [1.402, -0.299 * 1.402 / 0.587, 0.000],
-        ],
-        dtype=np.float32,
-    )
+    arr = arr.astype(np.float32)
+    arr -= [0, 2 ** (bit_depth - 1), 2 ** (bit_depth - 1)]
+
+    # Round(x) -> floor of (arr + 0.5)
+    np.matmul(arr, _YBR_FULL_TO_RGB, out=arr)
+    arr += 0.5
+    np.floor(arr, out=arr)
```

Comment:
is this different from `np.rint()`?

Reply:
Probably slightly different. The exact (as opposed to approximate) conversion in ITU T.871 requires a +0.5 prior to the floor operation.

Reply:
![image](https://github.com/user-attachments/assets/dc3e0d26-55fb-45d5-8a59-a682693657b9)